content/en/docs/tasks/configure-pod-container/security-context.md

```diff
@@ -52,10 +52,11 @@ Here is a configuration file for a Pod that has a `securityContext` and an `empt
 {{< codenew file="pods/security/security-context.yaml" >}}
 
 In the configuration file, the `runAsUser` field specifies that for any Containers in
-the Pod, the first process runs with user ID 1000. The `fsGroup` field specifies that
-group ID 2000 is associated with all Containers in the Pod. Group ID 2000 is also
-associated with the volume mounted at `/data/demo` and with any files created in that
-volume.
+the Pod, all processes runs with user ID 1000. The `runAsGroup` field specifies the primary group ID of 3000 for 
+all processes within any containers of the Pod. If this field is ommitted, the primary group ID of the containers 
+will be root(0). Any files created will also be owned by user 1000 and group 3000 when `runAsGroup` is specified. 
+If `fsGroup` field is specified, all processes of the container are also part of the supplementary group ID 2000. 
+Group ID 2000 is also associated with the volume mounted at `/data/demo` and with any files created in that volume. 
```

Comment:
@kubernetes/sig-storage-pr-reviews, PTAL at this description for accuracy

Reply:
To be more specific, the volume's owner is set to 2000.

Also, I'm not sure if this is the best place to mention it, but not all volume types implement fsgroup.  Generally single-writer volume types implement it, but not multi-writer volume types like nfs.

Reply:
The wording is a little confusing: ``If `fsGroup` field is specified, ...``. This is talking about a specific example, and `fsGroup` _is_ specified. Prefer:

> Since `fsGroup` is specified, all process of the container ...

Reply:
I think we should add that block in a storage related enhancement, i can add it if you can point me to the right doc @msau42

Reply:
We don't have a separate section for fsgroup in the volumes.md page.  Maybe we should add a section that lists all the drivers that support it and then link it from this page

Reply:
Sounds good , should i open an issue and assign to you @msau42  ?

Reply:
yes that's fine